catalogue/webapp/services/prismic/types/index.ts

```diff
@@ -52,8 +52,8 @@ type Promo = {
     image: {
       url: string;
       dimensions: ImageDimensions;
-      alt?: string;
-      copyright?: string;
+      alt: string | null;
+      copyright: string | null;
```

Comment:
Was getting an error that it didn't match `common/model/image.ts`'s `ImageType`. I'm hoping we can clean this up properly when moving to ES anyway.